src/common/Mutex.cc

```diff
@@ -13,10 +13,10 @@
  */
 
 #include "common/Mutex.h"
-#include "common/perf_counters.h"
 #include "common/config.h"
 #include "common/Clock.h"
 #include "common/valgrind.h"
+#include "common/ceph_context.h"
```

Comment:
why do we need to trade `perf_counters.h` with `ceph_context.h` ? i think we can just drop the `#include "common/perf_counters.h"` in this file, because of 03b519114e22cd91d2641cadfddf2024d1c2710d .